src/current/molt/migration-overview.md

```diff
@@ -0,0 +1,136 @@
+    <td>Schema and data validation</td>
+    <td>PostgreSQL 12-15, MySQL 5.7-8.0+, CockroachDB</td>
+    <td><a href="{% link {{ site.current_cloud_version }}/cockroachdb-feature-availability.md %}">Preview</a></td>
+  </tr>
+</table>
+
+### Schema Conversion Tool
+
+The [MOLT Schema Conversion Tool]({% link cockroachcloud/migrations-page.md %}) converts a source database schema to a CockroachDB-compatible schema. The tool:
+
+- Identifies [unimplemented features]({% link molt/migration-strategy.md %}#unimplemented-features-and-syntax-incompatibilities)
+- Rewrites unsupported [DDL syntax]({% link {{ site.current_cloud_version }}/sql-statements.md %}#data-definition-statements)
+- Applies CockroachDB [schema best practices]({% link molt/migration-strategy.md %}#schema-design-best-practices)
+
+### Fetch
+
+[MOLT Fetch]({% link molt/molt-fetch.md %}) performs the core data migration to CockroachDB. It supports:
+
+- [Multiple migration modes](#migration-modes) via `IMPORT INTO` or `COPY FROM`
+- Data movement via [cloud storage, local file servers, or direct copy]({% link molt/molt-fetch.md %}#data-path).
+- [Concurrent data export]({% link molt/molt-fetch.md %}#best-practices) from multiple source tables and shards
+- [Continuous replication]({% link molt/molt-fetch.md %}#replicate-changes), enabling you to minimize downtime before cutover
+- [Schema transformation rules]({% link molt/molt-fetch.md %}#transformations)
+- Safe [continuation]({% link molt/molt-fetch.md %}#fetch-continuation) to retry failed or interrupted tasks from specific checkpoints
+- [Failback]({% link molt/molt-fetch.md %}#fail-back-to-source-database) to replicate changes from CockroachDB back to the original source via a secure changefeed
+
+### Verify
+
+[MOLT Verify]({% link molt/molt-verify.md %}) checks for data and schema discrepancies between the source database and CockroachDB. It performs:
+
+- Table structure verification
+- Column definition verification
+- Row-level data verification
+{% comment %}- Continuous, live, or one-time verification{% endcomment %}
+
+## Migration modes
+
+MOLT Fetch supports [multiple data migration modes]({% link molt/molt-fetch.md %}#fetch-mode). These can be combined based on your testing and cutover strategy.
+
+|                     Mode                    |                                 Description                                  |                                               Best For                                               |
+|---------------------------------------------|------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| `--mode data-load`                          | Performs one-time load of source data into CockroachDB                       | Testing, migrations with planned downtime, [phased migrations]({% link molt/migrate-in-phases.md %}) |
```

Comment:
Maybe worth calling that this "planned downtime" is if your data load is so fast that it can fit within the window.

Otherwise you should do replication.